Branch-SDK-TestBed/AndroidManifest.xml

```diff
@@ -14,7 +14,7 @@
         android:allowBackup="true"
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
-        android:name="io.branch.referral.BranchApp" >
+        android:name="io.branch.branchandroiddemo.CustomBranchApp" >
```

Comment:
LeakCanary's default implementation requires an app-level hook (see the code for `CustomBranchApp`, it's pretty straightforward)